### ISSUE TITLE
Error running `npx @edgedb/generate edgeql-js`

### ISSUE THREAD

**Issue by zchenb** [edgedb/edgedb-js#753]
**Describe the bug**

Running `npx @edgedb/generate edgeql-js` results in the following error.
```
Generating query builder...
Detected package.json. Generating .js files with CommonJS module syntax.
   To override this, use the --target flag.
   Run `npx @edgedb/generate --help` for full options.
Introspecting database schema...
Cannot find 'anytype' in multirange<std::anypoint>
```

**Reproduction**
Create `default.esdl`
run `migration create/apply`
run the `npx @edgedb/generate edgeql-js` command

```
using extension auth;
using extension edgeql_http;

module default {


    type User {
        name: str;
        email: str;
        identity: ext::auth::Identity;
    }

}
```

**Expected behavior**
It should not happen

**Versions (please complete the following information):**

- OS: macOS
- EdgeDB version (e.g. `2.0`): "4.0-rc.1+0dfbf64
- EdgeDB CLI version (e.g. `2.0`): EdgeDB CLI 4.1.0-dev.1057+4a6d031
- `edgedb-js` version (e.g. `0.20.10`;): edgedb@1.3.7
- Node/Deno version: v20.5.0

**Additional context**

**Comment by KaelWD** [comment]
Not getting this anymore with
```
"@edgedb/generate": "^0.4.1",
"edgedb": "^1.4.0",
```

**Comment by scotttrinh** [comment]
Fixed in #733
